exercises/palindrome-products/PalindromeProductsExample.swift

```diff
@@ -0,0 +1,60 @@
+// Foundation not needed
+
+// Apple Swift version 2.1
+
+struct Palindromes {
+    
+    struct Palindrome {
```

Comment:
I don't see why this struct needs to be nested. My preference would be to only have 'Palindrome' as the main struct with largest and smallest as Int properties.